modules/custom/d_p_side_by_side/config/install/core.entity_form_display.paragraph.d_p_side_by_side.default.yml

```diff
@@ -2,6 +2,7 @@ langcode: en
 status: true
 dependencies:
   config:
+    - field.field.paragraph.d_p_side_by_side.field_d_main_title
```

Comment:
We should not have this field here, `side by side` contains only reference elements